### ISSUE TITLE
npmjs.com distribution still pulls in lodash

### ISSUE THREAD

**Issue by starpit** [caolan/async#1566]
**What version of async are you using?**
2.6.1

**Which environment did the issue occur in (Node version/browser version)**
i've tried both 8.11 and 10.7

**What did you do? Please include a minimal reproducable case illustrating issue.**
```
$ mkdir tmp && cd tmp
$ npm init -y && npm install async
$ ls node_modules
async  lodash
$ grep lodash node_modules/async/package.json
    "lodash": "^4.17.10"
```

whereas
```
$ git clone git@github.com:caolan/async.git
grep lodash async/package.json  | wc -l
       0
```

**What did you expect to happen?**
no lodash dependence when installed from npm

**What was the actual result?**
lodash still seems to be a dependence of the npmjs publication

**Comment by aearly** [comment]
We haven't yet published a 3.0.0 RC with lodash removed yet.

**Comment by starpit** [comment]
got it, thanks!
